### PR TITLE
ICMSLST-582 Add CSF view application view.

### PR DESCRIPTION
**Pull request by MattHolmes123**
Other changes:
  - Refactor CFS templates to have common prefix.

### CODE REVIEW

File: web/templates/web/domains/case/export/cfs-view.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  {% block application_contact %}
+    {{ super() }}
+  {% endblock %}
+
+  {% call application_section("Certificate of Free Sale") %}
+    {{ application_field(application_countries|nl2br, "Countries") }}
+  {% endcall %}
+
+  {% for schedule in schedules %}
+    {% set schedule_legislations = schedule.legislations.all().values_list("name", flat=True) %}
```
Comment (mikeylockwood) [inline web/templates/web/domains/case/export/cfs-view.html:32]: Shouldn't need `.all()` here

Comment (MattHolmes123) [reply web/templates/web/domains/case/export/cfs-view.html:32]: Changed.

**Review by kevinetienne — APPROVED** [review]

File: web/domains/case/views.py [hunk 2]
```diff
@@ -1467,6 +1470,25 @@ def _view_com(
...
+    cfs_fields = CFSSchedule._meta.get_fields()
+    labels = {f.name: getattr(f, "verbose_name", "") for f in cfs_fields}
+    app_countries = "\n".join(application.countries.all().values_list("name", flat=True))
+
+    context = {
+        "process_template": "web/domains/case/export/partials/process.html",
+        "process": application,
+        "labels": labels,
+        "application_countries": app_countries,
+        "schedules": application.schedules.filter(is_active=True),
```
Comment (mikeylockwood) [inline web/domains/case/views.py:1485]: Maybe an `order_by` to guarantee the ordering when looping through the schedules in the template

**Review by mikeylockwood — APPROVED** [review]

File: web/domains/case/views.py (hunk 2 above)
Comment (MattHolmes123) [reply web/domains/case/views.py:1485]: good shout.
Updated

**Pull request merged**